src/utils/supergraph_config.rs

```diff
@@ -466,6 +284,12 @@ mod test_get_supergraph_config {
             let graphref_raw = format!("{name}@{variant}");
             let url = format!("http://{}.remote.com", name);
             server.mock(|when, then| {
+                let request_body_partial = json!({
+                    "variables": {
+                        "graph_ref": graphref_raw,
+                    },
+                    "operationName": "SubgraphFetchAllQuery"
+                });
```

Comment:
This replaces the previous contents of L487 - 514. It seems that on Windows the line breaks that were contained within the query expression were causing the tests to trip up. There's no need for this because we can use another function of the Mock library to match on a partial JSON body, that still could only have been sent from the bit of the code we're testing.

A similar change is made in another test that fell foul of this issue.